cfgov/v1/jinja2/v1/doing-business-with-us/_template-opportunities-well-content.html

```diff
@@ -0,0 +1,71 @@
+<h2>Opportunities to work with us</h2>
```

Comment:
I'll remove `-content` too, `_template` implies it's some content of some sort already.